Jenkinsfile.integration

```diff
@@ -267,8 +267,13 @@ pipeline {
                 // Do not leave the ssh-agent running
                 sh "pkill -f 'ssh-agent -a /tmp/${env.SOCOK8S_ENVNAME}'"
                 if (fileExists("/tmp/${env.SOCOK8S_ENVNAME}.needcleanup")) {
-                    sh './run.sh teardown'
-                    sh 'rm /tmp/${SOCOK8S_ENVNAME}.needcleanup'
+                    try {
+                        sh './run.sh teardown'
+                    } catch (e) {
+                        sh 'ansible-playbook playbooks/generic-notify_failure_rocket.yml -v'
+                    } finally {
+                        sh 'rm /tmp/${SOCOK8S_ENVNAME}.needcleanup'
```

Comment:
I'd rather this stick around if the teardown fails. It gives us an alternate list of what might need cleaning up rather than rocketchat in case we miss messages.

(and worst case if we forget to remove the .needcleanup file after we have manually resolved something it's just an empty file in /tmp/).

Reply:
:+1:

Reply:
good point. That also remind me that we need to set the build status to failure as we are caching an error!

Reply:
Agreed!